src/features/defi/providers/univ2/components/UniV2Manager/Deposit/components/Approve.tsx

```diff
@@ -76,44 +97,43 @@ export const Approve: React.FC<FoxEthLpApproveProps> = ({ accountId, onNext }) =
   const toast = useToast()
 
   const handleApprove = useCallback(async () => {
-    if (!dispatch || !state?.deposit || !foxEthLpOpportunity || !wallet || !supportsETH(wallet))
-      return
+    if (!dispatch || !state?.deposit || !lpOpportunity || !wallet || !supportsETH(wallet)) return
 
     try {
-      dispatch({ type: FoxEthLpDepositActionType.SET_LOADING, payload: true })
+      dispatch({ type: UniV2DepositActionType.SET_LOADING, payload: true })
       await approve()
       await poll({
         fn: () => allowance(),
         validate: (result: string) => {
-          const allowance = bnOrZero(result).div(bn(10).pow(foxAsset.precision))
-          return bnOrZero(allowance).gte(bnOrZero(state.deposit.foxCryptoAmount))
+          const allowance = bnOrZero(result).div(bn(10).pow(asset1.precision))
+          return bnOrZero(allowance).gte(bnOrZero(state.deposit.asset1CryptoAmount))
         },
         interval: 15000,
         maxAttempts: 30,
       })
       // Get deposit gas estimate
       const gasData = await getDepositGasDataCryptoBaseUnit({
-        token0Amount: state.deposit.ethCryptoAmount,
-        token1Amount: state.deposit.foxCryptoAmount,
+        token0Amount: state.deposit.asset0CryptoAmount,
+        token1Amount: state.deposit.asset1CryptoAmount,
       })
       if (!gasData) return
       const estimatedGasCryptoPrecision = bnOrZero(gasData.average.txFee)
         .div(bn(10).pow(feeAsset.precision))
         .toPrecision()
       dispatch({
-        type: FoxEthLpDepositActionType.SET_DEPOSIT,
+        type: UniV2DepositActionType.SET_DEPOSIT,
         payload: { estimatedGasCryptoPrecision },
       })
 
       onNext(DefiStep.Confirm)
       trackOpportunityEvent(
         MixPanelEvents.DepositApprove,
         {
-          opportunity: foxEthLpOpportunity,
-          fiatAmounts: [state.deposit.foxFiatAmount, state.deposit.ethFiatAmount],
+          opportunity: lpOpportunity,
+          fiatAmounts: [state.deposit.asset1FiatAmount, state.deposit.asset0FiatAmount],
```

Comment:
why is this `[asset1, asset0]` ordering?

Reply:
Nice one, shouldn't matter since it's fiat amounts but definitely should be in the same order - 72b24f0cfeab0f30b200305f9c2a9786e9b0dfd3